incubator/elasticsearch/README.md

```diff
@@ -113,6 +113,9 @@ The following table lists the configurable parameters of the elasticsearch chart
 | `data.tolerations`                   | Data tolerations                                                    | `{}`                                 |
 | `data.terminationGracePeriodSeconds` | Data termination grace period (seconds)                             | `3600`                               |
 | `data.antiAffinity`                  | Data anti-affinity policy                                           | `soft`                               |
+| `plugins.command`                    | Path to Elasticsearchs plugin command                               | `"/usr/share/elasticsearch/bin/elasticsearch-plugin"` |
```

Comment:
I am still not sure about doing this.
This goes against the immutability of containers.